### PR TITLE
CW-653-Migrate-Tron-And-Solana-To-NowNodes

### PR DESCRIPTION
**Pull request by Blazebrain**
Issue Number (if Applicable): Fixes #

# Description

Please include a summary of the changes and which issue is fixed / feature is added. 

# Pull Request - Checklist  

- [ ] Initial Manual Tests Passed
- [ ] Double check modified code and verify it with the feature/task requirements
- [ ] Format code
- [ ] Look for code duplication
- [ ] Clear naming for variables and methods

### CODE REVIEW

File: cw_shared_external/pubspec.lock
Comment (OmarHatem28) [inline cw_shared_external/pubspec.lock:1]: remove this please

File: devtools_options.yaml
Comment (OmarHatem28) [inline devtools_options.yaml:1]: what's that?

Comment (Blazebrain) [reply devtools_options.yaml:1]: It's a file that's automatically created if you open devtools on a codebase.

I was trying to check the performance of the app and probably note down some places where things slow down a bit.

File: cw_shared_external/pubspec.lock
Comment (Blazebrain) [reply cw_shared_external/pubspec.lock:1]: Alright

**Review by OmarHatem28 — APPROVED** [review]

**Pull request merged**